Mathlib/Order/Bounds/Lattice.lean

```diff
@@ -0,0 +1,47 @@
+/-
+Copyright (c) 2024 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import Mathlib.Data.Set.Lattice
+
+/-!
+# Unions and intersections of bounds
+
+Some results about upper and lower bounds over collections of sets.
+
+## Implementation notes
+
+In a separate file as we need to import `Mathlib.Data.Set.Lattice`.
+
+-/
+
+variable {α : Type*} [Preorder α]
```

Comment:
Add this to these variables, to avoid duplicating them below? `{ι : Sort*} {s : ι → Set α}` (perhaps also u)